test/deps/import-busybox

```diff
@@ -316,6 +316,8 @@ user.foo: {{ config_get("user.foo", "_unset_") }}
 if __name__ == "__main__":
     if "LXD_DIR" in os.environ:
         lxd_socket = os.path.join(os.environ["LXD_DIR"], "unix.socket")
+    elif os.path.exists("/var/snap/lxd/common/lxd/unix.socket"):
```

Comment:
Whats ur thinking with this one, for a future plan regarding testing?

Reply:
Mostly usability for devs. This mimics `pyLXD` behaviour of trying to guess where the socket is.